plugin/main.cpp

```diff
@@ -19,7 +19,8 @@
 Q_DECL_EXPORT int main(int argc, char *argv[])
 {
     QLoggingCategory::setFilterRules(QStringLiteral("geoclue.provider.mlsdb.debug=false\n"
-                                                    "geoclue.provider.mlsdb.position.debug=false"));
+                                                    "geoclue.provider.mlsdb.position.debug=false\n"
+                                                    "geoclue.provider.mlsdb.online.debug=false"));
```

Comment:
What turns these on by default, all that are not false by default are on by default? So if we would not set anything false everything would be false by default or how does it go? Also if we set these explicitly off here can one turn those on from env variable anymore after this?